workspace-manager-clienttests/src/main/java/scripts/utils/CloudContextMaker.java

```diff
@@ -0,0 +1,47 @@
+package scripts.utils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.model.CloudPlatform;
+import bio.terra.workspace.model.CreateCloudContextRequest;
+import bio.terra.workspace.model.CreateCloudContextResult;
+import bio.terra.workspace.model.JobControl;
+import bio.terra.workspace.model.JobReport;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CloudContextMaker {
```

Comment:
this may not technically be a factory in the sense of Factory Pattern, so maker might be more accurate. https://www.tutorialspoint.com/design_pattern/factory_pattern.htm